### PR TITLE
planner_cspace: fix map update range

### LINKED ISSUE #671: planner_cspace: segmentation fault during navigation test
https://github.com/at-wat/neonavigation/actions/runs/4220279514/jobs/7326418864#step:9:6274
```
  [6731] Navigation success.
  [ERROR] [1676870809.816362591] [/planner_3d]: Position jumped (2.363/1.000, 1.557/1.500); clearing history
  [6731] 1676870758.780823249 Local map received.
  [6731] 1676870810.070750385 Path updated. (empty)
  [6731] 1676870758.208458151 Map received.
  [6731] 1676870758.208458151 Costmap received.
  [6731] 1676870809.581226346 Status updated.
    header: 1676870809.581226346 
    status: 0
    error: 0
  [6731] 1676870810.223644345 Map applied.
  [ WARN] [1676870810.232917013] [/planner_3d]: Forgetting remembered costmap.
  [ WARN] [1676870810.361336032] [/patrol]: Patrol finished. Waiting next path.
  [6731] 1676870758.208458151 Costmap received.
  [6731]  Local map applied.
  [6731] 1676870813.368815136 Status updated.
    header: 1676870813.368815136 
    status: 1
    error: 0
  [6731] 1676870813.378990220 Path updated.
  2.45, 0.45
  1.35, 2.55
  
  Thread 1 "planner_3d" received signal SIGSEGV, Segmentation fault.
  planner_cspace::CyclicVecBase<3, 2, int>::operator+<int> (v=..., this=0x7fff2e7ea048) at /catkin_ws/src/self/planner_cspace/include/planner_cspace/cyclic_vec.h:182
  182	      out[i] += v[i];
  #0  planner_cspace::CyclicVecBase<3, 2, int>::operator+<int> (v=..., this=0x7fff2e7ea048) at /catkin_ws/src/self/planner_cspace/include/planner_cspace/cyclic_vec.h:182
  #1  planner_cspace::planner_3d::DistanceMap::fillCostmap(planner_cspace::reservable_priority_queue<planner_cspace::GridAstar<3, 2>::PriorityVec>&, planner_cspace::CyclicVecBase<3, 2, int> const&) [clone ._omp_fn.0] () at /catkin_ws/src/self/planner_cspace/src/distance_map.cpp:122
  #2  0x00007f1cf64adedf in GOMP_parallel () from /usr/lib/x86_64-linux-gnu/libgomp.so.1
  #3  0x000055f6dc9407da in planner_cspace::planner_3d::DistanceMap::fillCostmap (this=this@entry=0x7fff2e7eb7a0, open=..., s_rough=...) at /catkin_ws/src/self/planner_cspace/src/distance_map.cpp:57
  #4  0x000055f6dc941f3a in planner_cspace::planner_3d::DistanceMap::update (this=0x7fff2e7eb7a0, s=..., e=..., rect=...) at /catkin_ws/src/self/planner_cspace/src/distance_map.cpp:302
  #5  0x000055f6dc8e21d8 in planner_cspace::planner_3d::Planner3dNode::cbMapUpdate (this=0x7fff2e7eae30, msg=...) at /catkin_ws/src/self/planner_cspace/src/planner_3d.cpp:838
  #6  0x000055f6dc8a8b7c in boost::_mfi::mf1<void, planner_cspace::planner_3d::Planner3dNode, boost::shared_ptr<costmap_cspace_msgs::CSpace3DUpdate_<std::allocator<void> > const> >::operator() (a1=..., p=<optimized out>, this=<optimized out>) at /usr/include/boost/bind/mem_fn_template.hpp:165
  #7  boost::_bi::list2<boost::_bi::value<planner_cspace::planner_3d::Planner3dNode*>, boost::arg<1> >::operator()<boost::_mfi::mf1<void, planner_cspace::planner_3d::Planner3dNode, boost::shared_ptr<costmap_cspace_msgs::CSpace3DUpdate_<std::allocator<void> > const> >, boost::_bi::rrlist1<boost::shared_ptr<costmap_cspace_msgs::CSpace3DUpdate_<std::allocator<void> > const> > > (a=<synthetic pointer>..., f=..., this=<optimized out>) at /usr/include/boost/bind/bind.hpp:319
  #8  boost::_bi::bind_t<void, boost::_mfi::mf1<void, planner_cspace::planner_3d::Planner3dNode, boost::shared_ptr<costmap_cspace_msgs::CSpace3DUpdate_<std::allocator<void> > const> >, boost::_bi::list2<boost::_bi::value<planner_cspace::planner_3d::Planner3dNode*>, boost::arg<1> > >::operator()<boost::shared_ptr<costmap_cspace_msgs::CSpace3DUpdate_<std::allocator<void> > const> > (a1=..., this=<optimized out>) at /usr/include/boost/bind/bind.hpp:1306
  #9  boost::detail::function::void_function_obj_invoker1<boost::_bi::bind_t<void, boost::_mfi::mf1<void, planner_cspace::planner_3d::Planner3dNode, boost::shared_ptr<costmap_cspace_msgs::CSpace3DUpdate_<std::allocator<void> > const> >, boost::_bi::list2<boost::_bi::value<planner_cspace::planner_3d::Planner3dNode*>, boost::arg<1> > >, void, boost::shared_ptr<costmap_cspace_msgs::CSpace3DUpdate_<std::allocator<void> > const> >::invoke (function_obj_ptr=..., a0=...) at /usr/include/boost/function/function_template.hpp:159
  #10 0x000055f6dc93e151 in boost::function1<void, boost::shared_ptr<costmap_cspace_msgs::CSpace3DUpdate_<std::allocator<void> > const> >::operator() (a0=..., this=0x55f6dcc50e58) at /usr/include/boost/function/function_template.hpp:759
  #11 ros::SubscriptionCallbackHelperT<boost::shared_ptr<costmap_cspace_msgs::CSpace3DUpdate_<std::allocator<void> > const>, void>::call (this=0x55f6dcc50e50, params=...) at /opt/ros/melodic/include/ros/subscription_callback_helper.h:144
  #12 0x00007f1cf7ddbe92 in ros::SubscriptionQueue::call() () from /opt/ros/melodic/lib/libroscpp.so
  #13 0x00007f1cf7d86559 in ros::CallbackQueue::callOneCB(ros::CallbackQueue::TLS*) () from /opt/ros/melodic/lib/libroscpp.so
  #14 0x00007f1cf7d882fb in ros::CallbackQueue::callAvailable(ros::WallDuration) () from /opt/ros/melodic/lib/libroscpp.so
  #15 0x00007f1cf7dc83ce in ros::spinOnce() () from /opt/ros/melodic/lib/libroscpp.so
  #16 0x000055f6dc914b32 in planner_cspace::planner_3d::Planner3dNode::waitUntil (previous_path=..., next_replan_time=..., this=<optimized out>) at /catkin_ws/src/self/planner_cspace/src/planner_3d.cpp:1227
  #17 planner_cspace::planner_3d::Planner3dNode::spin (this=0x7fff2e7eae30) at /catkin_ws/src/self/planner_cspace/src/planner_3d.cpp:1286
  #18 0x000055f6dc887442 in main (argc=<optimized out>, argv=0x7fff2e7ec178) at /catkin_ws/src/self/planner_cspace/src/planner_3d.cpp:1803
  [6731]  Local map applied.
  [6731]  Local map applied.
  [6731]  Local map applied.
  [6731]  Local map applied.
  [6731]  Local map applied.
  [6731]  Local map applied.
  [6731]  Local map applied.
  [6731]  Local map applied.
  [6731]  Local map applied.
  [6731]  Local map applied.
  [6731]  Local map applied.
  [6731]  Local map applied.
  [6731]  Local map applied.
  [6731]  Local map applied.
  [6731]  Local map applied.
  [6731]  Local map applied.
  [6731]  Local map applied.
  [6731]  Local map applied.
  [6731]  Local map applied.
  [6731]  Local map applied.
  [6731] 599 points recorded
  1676870811.534942123 2.5 0.45 3.14159
  1676870814.635021109 2.3934 0.459094 2.96943
  1676870815.735009967 2.28698 0.486494 2.81425
  1676870816.835045727 2.18483 0.527249 2.73942
  1676870817.935041286 2.0848 0.572989 2.69199
  1676870819.135043441 1.97732 0.626353 2.67308
  1676870820.335067596 1.87044 0.680911 2.66751
  1676870821.535051251 1.76324 0.734819 2.65658
  1676870822.735017023 1.65963 0.795298 2.5811
  1676870824.035043196 1.55284 0.869345 2.4833
  1676870825.435026566 1.44612 0.959913 2.41199
  1676870826.835036937 1.3437 1.05535 2.37457
  1676870828.334949503 1.23669 1.16046 2.35887
  1676870829.835038308 1.13063 1.26654 2.35444
  1676870831.335056240 1.02481 1.37285 2.35363
  1676870832.634958826 0.942068 1.47272 2.14813
  1676870833.734942669 0.89184 1.57035 1.93661
  1676870834.835023914 0.86466 1.67673 1.73891
  1676870835.935003157 0.850756 1.78581 1.65393
  1676870837.035041501 0.844949 1.89565 1.60581
  1676870838.134942406 0.850456 2.0053 1.41816
  1676870839.234929124 0.878323 2.11151 1.20726
  1676870840.135031978 0.918512 2.19187 1.00572
  1676870841.435042664 0.992661 2.29626 0.883626
  1676870842.834946131 1.0922 2.38752 0.68121
  1676870844.135013218 1.19666 2.46486 0.604586
  1676870845.434940002 1.30481 2.53698 0.577133
  [6731] 1676870758.208458151 Costmap received.
  [6731] 1676870758.208458151 Map received.
  [6731] 1676870758.780823249 Local map received.
  [6731] 1676870881.766023179 Map applied.
  [6731] 1676870758.208458151 Costmap received.
  ... logging to /root/.ros/log/rostest-1aba6e5ce865-6659.log
  [ROSUNIT] Outputting test results to /catkin_ws/build/test_results/planner_cspace/rostest-navigation_rostest__antialias_start_true__fast_map_update_true.xml
  [Testcase: testtest_navigate] ... ok
  
  [ROSTEST]-----------------------------------------------------------------------
  
  [planner_cspace.rosunit-test_navigate/Navigate][passed]
  [planner_cspace.rosunit-test_navigate/NavigateWithLocalMap][FAILURE]------------
  /catkin_ws/src/self/planner_cspace/test/src/test_navigate.cpp:373
  Failed
  [6731] Navigation timeout.
  now: 1676870871.435048402
  status: 
    header: 1676870813.378990220 
    status: 1
    error: 0
  --------------------------------------------------------------------------------
  
  [planner_cspace.rosunit-test_navigate/GlobalPlan][FAILURE]----------------------
  /catkin_ws/src/self/planner_cspace/test/src/test_navigate.cpp:451
  Value of: srv_plan.call(req, res)
    Actual: false
  Expected: true
  --------------------------------------------------------------------------------
```
[logs-melodic.tar.zip](https://github.com/at-wat/neonavigation/files/10782179/logs-melodic.tar.zip)

### PR DESCRIPTION
**Pull request by at-wat**
- Fix memory access violation
- Fix map_update_max to contain upper boundary edge pixels
  Word definitions:
    - max: contains the value
    - size: does not contain the value

---

Fix #671 
Running related tests multiple times at https://github.com/at-wat/neonavigation/pull/666 and segmentation fault is not occurred.

### CODE REVIEW

**Comment by at-wat** [comment]
## [345] PASSED on melodic

<details><summary>All tests passed</summary>


```
build/test_results/costmap_cspace/gtest-test_costmap_3d.xml: 22 tests
build/test_results/costmap_cspace/gtest-test_pointcloud_accumulator.xml: 4 tests
build/test_results/costmap_cspace/roslint-costmap_cspace.xml: 1 tests
build/test_results/joystick_interrupt/roslint-joystick_interrupt.xml: 1 tests
build/test_results/joystick_interrupt/rostest-test_test_joystick_interrupt_rostest.xml: 1 tests
build/test_results/joystick_interrupt/rosunit-test_joystick_interrupt.xml: 12 tests
build/test_results/map_organizer/roslint-map_organizer.xml: 1 tests
build/test_results/map_organizer/rostest-test_test_map_organizer_rostest.xml: 1 tests
build/test_results/map_organizer/rostest-test_test_pointcloud_to_maps_rostest.xml: 1 tests
build/test_results/map_organizer/rosunit-test_map_organizer.xml: 8 tests
build/test_results/map_organizer/rosunit-test_pointcloud_to_maps.xml: 2 tests
build/test_results/neonavigation_common/roslint-neonavigation_common.xml: 1 tests
build/test_results/neonavigation_common/rostest-test_test_compat_rostest.xml: 1 tests
build/test_results/neonavigation_common/rosunit-test_compat.xml: 6 tests
build/test_results/obj_to_pointcloud/roslint-obj_to_pointcloud.xml: 1 tests
build/test_results/obj_to_pointcloud/rostest-test_test_obj_to_pointcloud_rostest.xml: 1 tests
build/test_results/obj_to_pointcloud/rosunit-test_obj_to_pointcloud.xml: 2 tests
build/test_results/planner_cspace/gtest-test_blockmem_gridmap.xml: 10 tests
build/test_results/planner_cspace/gtest-test_costmap_bbf.xml: 4 tests
build/test_results/planner_cspace/gtest-test_cyclic_vec.xml: 14 tests
build/test_results/planner_cspace/gtest-test_distance_map.xml: 14 tests
build/test_results/planner_cspace/gtest-test_grid_astar.xml: 12 tests
build/test_results/planner_cspace/gtest-test_grid_metric_converter.xml: 4 tests
build/test_results/planner_cspace/gtest-test_motion_cache.xml: 2 tests
build/test_results/planner_cspace/gtest-test_motion_primitive_builder.xml: 2 tests
build/test_results/planner_cspace/gtest-test_path_interpolator.xml: 4 tests
build/test_results/planner_cspace/gtest-test_planner_3d_cost.xml: 2 tests
build/test_results/planner_cspace/roslint-planner_cspace.xml: 1 tests
build/test_results/planner_cspace/rostest-navigation_rostest__antialias_start_true.xml: 1 tests
build/test_results/planner_cspace/rostest-navigation_rostest__antialias_start_true__fast_map_update_true.xml: 1 tests
build/test_results/planner_cspace/rostest-navigation_rostest__with_tolerance_true.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_abort_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_costmap_watchdog_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_debug_outputs_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_boundary_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_compat_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_planner_2dof_serial_joints_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_planner_3d_map_size_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_preempt_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_tolerant_action_rostest.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_abort.xml: 2 tests
build/test_results/planner_cspace/rosunit-test_costmap_watchdog.xml: 4 tests
build/test_results/planner_cspace/rosunit-test_debug_outputs.xml: 6 tests
build/test_results/planner_cspace/rosunit-test_navigate.xml: 6 tests
build/test_results/planner_cspace/rosunit-test_navigate_boundary.xml: 2 tests
build/test_results/planner_cspace/rosunit-test_planner_2dof_serial_joints.xml: 2 tests
build/test_results/planner_cspace/rosunit-test_planner_3d_map_size.xml: 12 tests
build/test_results/planner_cspace/rosunit-test_preempt.xml: 2 tests
build/test_results/planner_cspace/rosunit-test_tolerant_action.xml: 2 tests
build/test_results/safety_limiter/roslint-safety_limiter.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter2_rostest.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter_compat_rostest.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter_rostest.xml: 1 tests
build/test_results/safety_limiter/rosunit-test_safety_limiter.xml: 22 tests
build/test_results/safety_limiter/rosunit-test_safety_limiter2.xml: 2 tests
build/test_results/track_odometry/gtest-test_tf_projection.xml: 2 tests
build/test_results/track_odometry/roslint-track_odometry.xml: 1 tests
build/test_results/track_odometry/rostest-test_test_tf_projection_rostest.xml: 1 tests
build/test_results/track_odometry/rostest-test_test_track_odometry_rostest.xml: 1 tests
build/test_results/track_odometry/rosunit-test_tf_projection_node.xml: 8 tests
build/test_results/track_odometry/rosunit-test_track_odometry.xml: 10 tests
build/test_results/trajectory_tracker/gtest-test_trajectory_tracker_filter.xml: 6 tests
build/test_results/trajectory_tracker/gtest-test_trajectory_tracker_path2d.xml: 8 tests
build/test_results/trajectory_tracker/roslint-trajectory_tracker.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_recorder_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_tracker_overshoot_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_tracker_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_tracker_with_odom_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rostest-trajectory_tracker_rostest__odom_delay_0.xml: 1 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_recorder.xml: 2 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_tracker.xml: 16 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_tracker_overshoot.xml: 12 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_tracker_with_odom.xml: 4 tests
build/test_results/trajectory_tracker_msgs/gtest-test_path_with_velocity_conversion.xml: 4 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 296 tests, 0 errors, 0 failures, 0 skipped
```
</details>

File: planner_cspace/src/planner_3d.cpp [hunk 1]
```diff
@@ -965,10 +978,10 @@ class Planner3dNode
     cm_base_ = cm_;
     bbf_costmap_.clear();
 
-    prev_map_update_x_min_ = std::numeric_limits<int>::max();
-    prev_map_update_x_max_ = std::numeric_limits<int>::lowest();
-    prev_map_update_y_min_ = std::numeric_limits<int>::max();
-    prev_map_update_y_max_ = std::numeric_limits<int>::lowest();
+    prev_map_update_x_min_ = map_info_.width;
+    prev_map_update_x_max_ = 0;
+    prev_map_update_y_min_ = map_info_.width;
```
Comment (Taka-Kazu) [inline planner_cspace/src/planner_3d.cpp:983]:
```suggestion
    prev_map_update_y_min_ = map_info_.height;
```

**Review by Taka-Kazu — APPROVED** [review]
LGTM

**Comment by at-wat** [comment]
## [346] FAILED on melodic

<details><summary>Test failed</summary>


```
build/test_results/costmap_cspace/gtest-test_costmap_3d.xml: 22 tests
build/test_results/costmap_cspace/gtest-test_pointcloud_accumulator.xml: 4 tests
build/test_results/costmap_cspace/roslint-costmap_cspace.xml: 1 tests
build/test_results/joystick_interrupt/roslint-joystick_interrupt.xml: 1 tests
build/test_results/joystick_interrupt/rostest-test_test_joystick_interrupt_rostest.xml: 1 tests
build/test_results/joystick_interrupt/rosunit-test_joystick_interrupt.xml: 12 tests
build/test_results/map_organizer/roslint-map_organizer.xml: 1 tests
build/test_results/map_organizer/rostest-test_test_map_organizer_rostest.xml: 1 tests
build/test_results/map_organizer/rostest-test_test_pointcloud_to_maps_rostest.xml: 1 tests
build/test_results/map_organizer/rosunit-test_map_organizer.xml: 8 tests
build/test_results/map_organizer/rosunit-test_pointcloud_to_maps.xml: 2 tests
build/test_results/neonavigation_common/roslint-neonavigation_common.xml: 1 tests
build/test_results/neonavigation_common/rostest-test_test_compat_rostest.xml: 1 tests
build/test_results/neonavigation_common/rosunit-test_compat.xml: 6 tests
build/test_results/obj_to_pointcloud/roslint-obj_to_pointcloud.xml: 1 tests
build/test_results/obj_to_pointcloud/rostest-test_test_obj_to_pointcloud_rostest.xml: 1 tests
build/test_results/obj_to_pointcloud/rosunit-test_obj_to_pointcloud.xml: 2 tests
build/test_results/planner_cspace/gtest-test_blockmem_gridmap.xml: 10 tests
build/test_results/planner_cspace/gtest-test_costmap_bbf.xml: 4 tests
build/test_results/planner_cspace/gtest-test_cyclic_vec.xml: 14 tests
build/test_results/planner_cspace/gtest-test_distance_map.xml: 14 tests
build/test_results/planner_cspace/gtest-test_grid_astar.xml: 12 tests
build/test_results/planner_cspace/gtest-test_grid_metric_converter.xml: 4 tests
build/test_results/planner_cspace/gtest-test_motion_cache.xml: 2 tests
build/test_results/planner_cspace/gtest-test_motion_primitive_builder.xml: 2 tests
build/test_results/planner_cspace/gtest-test_path_interpolator.xml: 4 tests
build/test_results/planner_cspace/gtest-test_planner_3d_cost.xml: 2 tests
build/test_results/planner_cspace/roslint-planner_cspace.xml: 1 tests
build/test_results/planner_cspace/rostest-navigation_rostest__antialias_start_true.xml: 1 tests
build/test_results/planner_cspace/rostest-navigation_rostest__antialias_start_true__fast_map_update_true.xml: 1 tests
build/test_results/planner_cspace/rostest-navigation_rostest__with_tolerance_true.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_abort_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_costmap_watchdog_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_debug_outputs_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_boundary_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_compat_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_planner_2dof_serial_joints_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_planner_3d_map_size_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_preempt_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_tolerant_action_rostest.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_abort.xml: 2 tests
build/test_results/planner_cspace/rosunit-test_costmap_watchdog.xml: 4 tests
build/test_results/planner_cspace/rosunit-test_debug_outputs.xml: 6 tests
build/test_results/planner_cspace/rosunit-test_navigate.xml: 6 tests
build/test_results/planner_cspace/rosunit-test_navigate_boundary.xml: 2 tests
build/test_results/planner_cspace/rosunit-test_planner_2dof_serial_joints.xml: 2 tests
build/test_results/planner_cspace/rosunit-test_planner_3d_map_size.xml: 12 tests
build/test_results/planner_cspace/rosunit-test_preempt.xml: 2 tests
build/test_results/planner_cspace/rosunit-test_tolerant_action.xml: 2 tests
build/test_results/safety_limiter/roslint-safety_limiter.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter2_rostest.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter_compat_rostest.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter_rostest.xml: 1 tests
build/test_results/safety_limiter/rosunit-test_safety_limiter.xml: 22 tests
build/test_results/safety_limiter/rosunit-test_safety_limiter2.xml: 2 tests
build/test_results/track_odometry/gtest-test_tf_projection.xml: 2 tests
build/test_results/track_odometry/roslint-track_odometry.xml: 1 tests
build/test_results/track_odometry/rostest-test_test_tf_projection_rostest.xml: 1 tests
build/test_results/track_odometry/rostest-test_test_track_odometry_rostest.xml: 1 tests
build/test_results/track_odometry/rosunit-test_tf_projection_node.xml: 8 tests
build/test_results/track_odometry/rosunit-test_track_odometry.xml: 10 tests
build/test_results/trajectory_tracker/gtest-test_trajectory_tracker_filter.xml: 6 tests
build/test_results/trajectory_tracker/gtest-test_trajectory_tracker_path2d.xml: 8 tests
build/test_results/trajectory_tracker/roslint-trajectory_tracker.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_recorder_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_tracker_overshoot_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_tracker_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_tracker_with_odom_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rostest-trajectory_tracker_rostest__odom_delay_0.xml: 1 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_recorder.xml: 2 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_tracker.xml: 16 tests, 0 errors, 2 failures, 0 skipped
build/test_results/trajectory_tracker/rosunit-test_trajectory_tracker_overshoot.xml: 12 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_tracker_with_odom.xml: 4 tests
build/test_results/trajectory_tracker_msgs/gtest-test_path_with_velocity_conversion.xml: 4 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 296 tests, 0 errors, 2 failures, 0 skipped
```
</details>

**Pull request merged**